pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -605,6 +605,19 @@ self: super: {
     '';
   });
 
+  d-bus = let
+    # Unreleased version for reasons...
+    newer = overrideSrc super.d-bus {
+      src = final.fetchFromGitHub {
+        owner = "Philonous";
+        repo = "d-bus";
+        rev = "fb8a948a3b9d51db618454328dbe18fb1f313c70";
+        hash = "sha256-R7/+okb6t9DAkPVUV70QdYJW8vRcvBdz4zKJT13jb3A=";
+      };
+    };
+  # Add now required extension on recent compilers.
+  in appendPatch newer ./patches/d-bus-Required-extension-for-ghc-8.10.1-and-up.patch;
```

Comment:
Shouldn‘t this patch exist as a PR to the project? We very much prefer pulling the patch in via fetchPatch rather than accumulating patches in the nixpkgs repo.

Reply:
I've opened an MR upstream with this and other fixes. I have added the link as a comment.